inference-engine/src/low_precision_transformations/src/low_precision.cpp

```diff
@@ -283,3 +284,24 @@ bool ngraph::pass::low_precision::LowPrecision::isFunctionQuantized(const std::s
     }
     return false;
 }
+
+bool ngraph::pass::low_precision::LowPrecision::isFQLevelsPresent(
```

Comment:
we have method for iteration over nGraph function to check `FakeQuantize` operations: `LowPrecision::isFunctionQuantized`. Instead new iteration over nGraph function, can we extend the method `LowPrecision::isFunctionQuantized`: return `FakeQuantize` existing quantization levels and reuse result?